File: tests/fixtures/local-dev-samples-legacy/python-deps/handler.py
import json
import requests

def handler(event):
    # Verify we can import requests
    version = requests.__version__
    
    return {
        "status": 200,
        "headers": {"Content-Type": "application/json"},
        "body": json.dumps({
            "message": "Hello from Python with Deps!",
            "requests_version": version
        })
    }

File: tests/fixtures/local-dev-samples-migrated/node-auto-deps/handler.js
const isOdd = require('is-odd');

module.exports.handler = async (event, context) => {
  return {
    status: 200,
    headers: {
      "Content-Type": "application/json"
    },
    body: JSON.stringify({
      message: "Hello from Node Auto Deps!",
      is_odd: isOdd(3)
    })
  };
};
